clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleNamedAccountCredentials.groovy

```diff
@@ -64,6 +64,7 @@ class GoogleNamedAccountCredentials implements AccountCredentials<GoogleCredenti
   final ConsulConfig consulConfig
   final Compute compute
   final String userDataFile
+  final List<String> regionsToIndex
```

Comment:
Should this still be `regionsToIndex`, since you're going to constrain mutating calls as well?